### DISCUSSION TITLE
SCA CONFIGURATION ISSUE

### DISCUSSION THREAD

**Discussion by forzaxhx** [wazuh/wazuh#17674] [General]
Hi. I encountered a problem configuring the SCA module. Our servers are configured in Spanish language and all checking rules are configured in English. Are there any official troubleshooting besides changing manually all checking rules?

**Comment by mdiego92** [comment]
Hello @forzaxhx,

Unfortunately, we do not provide full support for any language on any operating system and it is not always possible to build a rule that does not use the English language. 

The only option is to modify the OS-specific YML file to use the correct Spanish expressions in each check of the SCA. Please bear in mind that you need to consider checking special characters depending on each language.

Best regards!
